WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml

```diff
@@ -43,15 +43,31 @@
                 <Image Width="120" Source="avares://WalletWasabi.Fluent/Assets/TechnologyLogos/payjoin.png" ToolTip.Tip="{Binding PayJoinUrl}" HorizontalAlignment="Left" Margin="0 8 0 0" />
             </c:PreviewItem>
             <Separator IsVisible="{Binding IsPayJoin}" />
-            <c:PreviewItem Icon="{StaticResource timer_regular}"
-                           Text="Bitcoin miners will work hard to confirm your transaction within">
-              <TextBox Classes="selectableTextBlock" Text="{Binding ConfirmationTimeText, FallbackValue=~20 minutes }" />
-            </c:PreviewItem>
-            <Separator />
-            <c:PreviewItem Icon="{StaticResource paper_cash_regular}"
-                           Text="for an additional charge of">
-              <TextBox Classes="selectableTextBlock" Text="{Binding FeeText, FallbackValue=235 sats (≈55.34 USD)}" />
+
+            <c:PreviewItem Icon="{StaticResource incognito_filled}"
+                           Text="You will receive change from this transaction."
+                           IsVisible="{Binding TransactionHasChange}">
+              <Button Classes="activeHyperLink" Content="View suggestions to improve privacy by avoiding change." HorizontalAlignment="Left" Command="{Binding AvoidChangeCommand}"/>
```

Comment:
```suggestion
              <Button Classes="activeHyperLink" Content="Change the payment amount to improve privacy by avoiding change." HorizontalAlignment="Left" Command="{Binding AvoidChangeCommand}"/>
```

Same as the suggestion above [change the fee], it should start by saying what is done. 

Other than that, tACK 6be378f